src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -308,10 +308,12 @@ public int compare(FeedbackResponseAttributes o1, FeedbackResponseAttributes o2)
                 return order;
             }
 
-            String receiverTeam1 = getTeamNameForEmail(o1.recipientEmail).isEmpty() ? getNameForEmail(o1.recipientEmail)
-                                                                                    : getTeamNameForEmail(o1.recipientEmail);
-            String receiverTeam2 = getTeamNameForEmail(o2.recipientEmail).isEmpty() ? getNameForEmail(o2.recipientEmail)
-                                                                                    : getTeamNameForEmail(o2.recipientEmail);
+            String receiverTeam1 = getTeamNameForEmail(o1.recipientEmail).isEmpty()
+                                 ? getNameForEmail(o1.recipientEmail)
+                                 : getTeamNameForEmail(o1.recipientEmail);
+            String receiverTeam2 = getTeamNameForEmail(o2.recipientEmail).isEmpty()
+                                 ? getNameForEmail(o2.recipientEmail)
+                                 : getTeamNameForEmail(o2.recipientEmail);
```

Comment:
Is there an accepted convention on line wrapping ternary operator? I don't feel this is right because the three symbols currently aligned are not at the same level. Furthermore, the first line gives a misleading impression (i.e. string being equated to a boolean).